man/org.freedesktop.systemd1.xml

```diff
@@ -1765,6 +1769,10 @@ node /org/freedesktop/systemd1 {
       Note that only the "innermost" virtualization technology is exported here. This detects both
       full-machine virtualizations (VMs) and shared-kernel virtualization (containers).</para>
 
+      <para><varname>ConfidentialVirtualization</varname> contains a short ID string describing the confidential
+      virtualization technology the system runs in. On bare-metal hardware this is the empty string. Otherwise,
+      it contains an identifier such as <literal>sev-snp</literal>, <literal>tdx</literal> and so on.</para>
```

Comment:
The doc here should list explicitly all identifiers that are currently supported